README.md

```diff
@@ -1,35 +1,21 @@
-+ [![All the Mods 3 Lite](http://cf.way2muchnoise.eu/274129.svg "ATM3: Lite") All The Mods 3 Lite](https://minecraft.curseforge.com/projects/atm-3-lite)  
-+ [![All the Mods Expert Remastered](http://cf.way2muchnoise.eu/274849.svg "ATMER") All The Mods Expert Remastered](https://mods.curse.com/modpacks/minecraft/252034-all-the-mods-expert)  
-  
-Need Help?
-======
-When reporting an issue put the version number before the issue title! Such as [FULL][1.37] My game is broken! Also include any added mods you may have put in, into the description of the issue. 
- 
-You can also find us on Discord for help or just to chat as well as Reddit:   
-  
-|              |
-|:------------:|
-|<a href="https://discordapp.com/invite/rbSZNDQ"><img src="https://discordapp.com/assets/fc0b01fe10a0b8c602fb0106d8189d9b.png" alt="Join us on Discord!"  width="200" height="68"></a>|
-|<a href="https://www.reddit.com/r/allthemods"><img src="https://www.redditstatic.com/about/assets/reddit-logo.png" alt="/r/AllTheMods on Reddit"  width="200" height="67"></a>|
-<br>
-<br>
-
-### Money / Donation
-
-All The Mods has never been about making money, we just love playing games and making cool modpacks and contributing to the modded community. We understand that sometimes people like to "say thanks" so by request we have setup a Patreon with some perks and are considering expanding some ATM branded merchandise as well. Any money that we do get is used to "give back" such as hosting free public ATM servers   
-
-<a href="https://www.patreon.com/allthemods">
-	<img src="https://c5.patreon.com/external/logo/become_a_patron_button.png" alt="Support us on Patreon"  width="200" height="47">
-</a>  
+[![Logo Image](https://i.imgur.com/R3RgVCP.png)](https://minecraft.curseforge.com/members/ATMTeam/projects)
 
+[![CurseForge](http://cf.way2muchnoise.eu/full_269708_downloads.svg)](https://minecraft.curseforge.com/projects/all-the-mods-3) [![Discord](https://img.shields.io/discord/254530689225981953.svg?&label=discord)](https://discordapp.com/invite/rbSZNDQ) [![Subreddit](https://img.shields.io/badge/subreddit-r%2Fallthemods-red.svg)](https://reddit.com/r/allthemods) [![GitHub Contributors](https://img.shields.io/github/contributors/AllTheMods/ATM-3.svg)](https://github.com/AllTheMods/ATM-3/graphs/contributors) [![GitHub Issues](https://img.shields.io/github/issues/AllTheMods/ATM-3.svg)](https://github.com/AllTheMods/ATM-3/issues)
+
+# All The Mods 3
+**All the Mods** started out as a private pack for just a few friends of mine that turned into something others wanted to play!
+
+It has all the basics that most other "big name" packs include but with a nice mix of some of newer or lesser-known mods as well.
+
+For example; Thaumcraft, Ender IO, Thermal Expansion for the big names and Rustic, Traverse and FastWorkBench for the not as well known!
+
+Can you get to the Creative items by making the "**ATM Star**"?
```

Comment:
Why is ATM Star in quotes?

Reply:
You’ll notice that all the descriptions were ripped off of CurseForge.